monai/apps/auto3dseg/auto_runner.py

```diff
@@ -569,58 +575,6 @@ def set_nni_search_space(self, search_space):
         self.search_space = search_space
         self.hpo_tasks = value_combinations
 
-    def set_image_save_transform(self, kwargs):
```

Comment:
The deletion of the methods:
- `set_image_save_transform`
- `set_ensemble_method`
mean breaking changes.

Reply:
Also I think `set_ensemble_method` is necessary, or otherwise, the user cannot change ensemble method in `AutoRunenr`.